python/ray/tests/test_core_worker_io_thread_stack_size.py

```diff
@@ -0,0 +1,27 @@
+import os
+import sys
+import pytest
+import ray
+
+
+@pytest.mark.asyncio
+async def test_core_worker_io_thread_stack_size(shutdown_only):
+    # Test to make sure core worker io thread
+    # has big enough stack size to run python code.
+    # See https://github.com/ray-project/ray/issues/41094 for more details.
+    ray.init()
+
+    @ray.remote(num_cpus=0)
+    def task():
+        return 1
+
+    assert (await task.remote()) == 1
```

Comment:
Maybe the task should try to import numpy or otherwise do some activity that requires a lot of stack?